boards/atxmega-a1u-xpro/include/eui_provider_params.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2021 Gerson Fernando Budke
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_atxmega-a1u-xpro
+ * @{
+ *
+ * @file
+ * @brief       EUI providers found on the board
+ *
+ * @author      Gerson Fernando Budke <nandojve@gmail.com>
+ */
+#ifndef EUI_PROVIDER_PARAMS_H
+#define EUI_PROVIDER_PARAMS_H
+
+#include <string.h>
+
+#include "net/eui64.h"
+#include "periph/cpuid.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief    CPUID provides a EUI-64, the same that is printed on the board
```

Comment:
I cannot find the CPU-ID. Do I understand correctly that it is printend on the silkscreen of the boards PCB?